src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,42 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class HomePage extends BasePage{
+    By searchField = By.cssSelector("[type='search']");
+    By viewAllButton = By.cssSelector("[data-test='view-all-songs-btn']");
+
+    By successNotification = By.cssSelector(".success.show");
+    By searchResults = By.cssSelector("#songResultsWrapper .song-item");
+    By avatarIcon = By.cssSelector("a .avatar");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```